### PR TITLE
Make landing page less ugly

### PR DESCRIPTION
**Pull request by StykMartin**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 
+import './App.css';
```
Comment (renanrodrigo) [inline src/App.tsx:5]: I guess its best to import patternfly css first, and then the custom one

File: src/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Beaker-Next</title>
+    <title>Dandelion</title>
```
Comment (renanrodrigo) [inline src/index.html:6]: Good catch. Could you also change package[-lock].json and the dockerfile to remove the beaker name?

Comment (StykMartin) [reply src/index.html:6]: All set

File: src/App.tsx (hunk 1 above)
Comment (StykMartin) [reply src/App.tsx]: Sure. Sounds good.

File: src/components/landingPage/LandingPage.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ import {
 
 const LandingPage: React.FC = () => {
   return (
-    <PageSection variant={PageSectionVariants.darker}>
+    <PageSection variant={PageSectionVariants.light}>
```
Comment (renanrodrigo) [inline src/components/landingPage/LandingPage.tsx:11]: hehe

**Pull request merged**